api/app/controllers/spree/api/v1/products_controller.rb

```diff
@@ -98,6 +96,10 @@ def product_params
             params.require(:product).permit(permitted_product_attributes)
           end
 
+          def find_product
```

Comment:
All private methods are indented with 2 extra spaces so just maintaining the consistency.